models/staging/stg_dbt_artifacts.sql

```diff
@@ -1,7 +1,6 @@
 with base as (
 
-    select *
-    from {{ source('dbt_artifacts', 'artifacts') }}
+    select * from {{ source('dbt_artifacts', 'artifacts') }}
```

Comment:
further down this model theres a `qualify index = 1` clause which weve probably never used before - can you add a comment explaining what thats doing